src/_assets/js/subscribe_form.js

```diff
@@ -0,0 +1,83 @@
+(function() {
+    // Form elements
+    const subscribeContainer = document.querySelector('.subscribe-container');
+    const subscribeForm = document.querySelector('.subscribe-container__form');
```

Comment:
If possible, please `querySelector` from the container (`subscribeContainer`), which is usually faster and reduces the chances for DOM conflicts.

Reply:
You can also document that this allows only for one instance of the form per page.